app.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "nzsl-online",
+  "stack": "heroku-18",
```

Comment:
can't decide if this is good practise.
I've been pressing the "upgrade stack" button on my apps. Staging and then prod.

Reply:
It is more explicit and dev-opsy. But it also can tie us to old ways in the future if we don't manage it.